GoogleApiWrappers/src/Google.Apis.Storage.v1.ClientWrapper/DownloadObjectOptions.cs

```diff
@@ -25,5 +31,26 @@ internal void ModifyDownloader(MediaDownloader downloader)
                 downloader.ChunkSize = ChunkSize.Value;
             }
         }
+
+        /// <summary>
+        /// Appends any parameters to the given builder, which is guaranteed to
+        /// already have one query parameter (alt=media).
+        /// </summary>
+        internal void AppendUriParameters(StringBuilder uriBuilder)
+        {
+            MaybeAppendParameter(uriBuilder, "generation", Generation);
+        }
+
+        private static void MaybeAppendParameter(StringBuilder uriBuilder, string name, long? value)
+        {
+            if (value != null)
+            {
+                uriBuilder
+                    .Append("&")
+                    .Append(name)
```

Comment:
It's a private method only called within this class, and the name will always be a constant - I think we're safe enough.